courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/2_dataset_api.ipynb

```diff
@@ -140,8 +140,7 @@
     "for i, (x, y) in enumerate(dataset):\n",
```

Comment:
<p>Same as in the lab. Set epochs=EPOCH in <code>create_dataset</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>